mikeio1d/geometry/geopandas/__init__.py

```diff
@@ -0,0 +1,18 @@
+from mikeio1d.various import try_import_geopandas
+from mikeio1d.various import try_import_shapely
+
+# check if geopandas and shapely are available
+try_import_geopandas()
```

Comment:
Would it makes sense to document here that MIKE IO 1D has a loose dependency on GeoPandas and it is required only if the functionality is used?